pacai/agents/learning/reinforcement.py

```diff
@@ -17,7 +17,7 @@ class ReinforcementAgent(ValueEstimationAgent):
         - Use self.getLegalActions(state) to know which actions
             are available in a state
     """
-
+    logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.DEBUG)
```

Comment:
Four things:
1) This line should be in the init of this file, we never want lines floating outside functions
2) Why are we setting the DEBUG level here? We want the basic level of logging to be set to INFO and if the students want more output or information they will pass in a flag that is DEBUG (which of course would then change the basic config level to DEBUG.
3) Lastly, we should only be needing to change the debug level in the files that have a main. I.E. any files that sit inside bin (this way we do not need to add this line to every file).
4) We need to have an improved formatting scheme: we want (logginglevel) (time of output) (message), 

I will again say "config issue" for all other cases of this.